### PR TITLE
SEOImages: User Story 1463880, Part 10

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Fixes #1463880

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw Here's the second PR for review. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019031400054/Report/.

File: docs/framework/wpf/advanced/advanced-text-formatting.md [hunk 1]
```diff
@@ -36,8 +36,7 @@ The Windows Presentation Foundation (WPF) provides a robust set of [!INCLUDE[TLA
   
  Unlike a traditional text [!INCLUDE[TLA#tla_api](../../../../includes/tlasharptla-api-md.md)], the <xref:System.Windows.Media.TextFormatting.TextFormatter> interacts with a text layout client through a set of callback methods. It requires the client to provide these methods in an implementation of the <xref:System.Windows.Media.TextFormatting.TextSource> class. The following diagram illustrates the text layout interaction between the client application and <xref:System.Windows.Media.TextFormatting.TextFormatter>.  
   
- ![Diagram of text layout client and TextFormatter](./media/textformatter01.png "TextFormatter01")  
-Interaction between application and TextFormatter  
+ ![Diagram of text layout client and TextFormatter](./media/advanced-text-formatting/text-layout-text-formatter-interaction.png)
```
Comment (mairaw) [inline docs/framework/wpf/advanced/advanced-text-formatting.md]: Perhaps this file name could have the name of the class all together?
```suggestion
 ![Diagram of text layout client and TextFormatter](./media/advanced-text-formatting/text-layout-textformatter-interaction.png)  
```

File: docs/framework/wpf/advanced/drawing-formatted-text.md [hunk 2]
```diff
@@ -56,35 +54,30 @@ Displayed text showing wordwrapping and ellipsis
 ### Converting Formatted Text to a Geometry  
  You can convert formatted text into <xref:System.Windows.Media.Geometry> objects, allowing you to create other types of visually interesting text. For example, you could create a <xref:System.Windows.Media.Geometry> object based on the outline of a text string.  
   
- ![Text outline using a linear gradient brush](./media/outlinedtext02.jpg "OutlinedText02")  
-Text outline using a linear gradient brush  
+ ![Text outline using a linear gradient brush](./media/typography-in-wpf/text-outline-linear-gradient.jpg)
```
Comment (mairaw) [inline docs/framework/wpf/advanced/drawing-formatted-text.md]: both articles were using the same image?

File: docs/framework/wpf/advanced/typography-in-wpf.md [hunk 3]
```diff
@@ -12,13 +12,12 @@ This topic introduces the major typographic features of [!INCLUDE[TLA2#tla_wincl
...
   
- ![Text with ClearType y&#45;direction anti&#45;aliasing](./media/typographyinwpf02.gif "TypographyInWPF02")  
+ ![Text with ClearType y&#45;direction anti&#45;aliasing](./media/typography-in-wpf/text-y-direction-antialiasing.gif)
```
Comment (mairaw) [inline docs/framework/wpf/advanced/typography-in-wpf.md]:
```suggestion
 ![Text with ClearType y-direction anti-aliasing](./media/typography-in-wpf/text-y-direction-antialiasing.gif)  
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Looks great @damabe. Left a few comments.

File: docs/framework/wpf/advanced/drawing-formatted-text.md (hunk 2 above)
Comment (damabe) [reply docs/framework/wpf/advanced/drawing-formatted-text.md]: Yes. There were a few images used across a couple of articles.

**Comment by damabe** [comment]
@mairaw I've made the changes and responded to your question. New CATS after the changes: https://contentqacats.azurewebsites.net/CheckResults/2019031500031/Report/.

File: docs/framework/wpf/advanced/drawing-formatted-text.md (hunk 2 above)
Comment (mairaw) [reply docs/framework/wpf/advanced/drawing-formatted-text.md]: Ok.

**Review by mairaw — APPROVED** [review]
Thanks @damabe!

**Pull request merged**